Adding new PNI nav

gvn opened the pull request:
related to #497

gvn commented:
@beccaklam Can you give this a design review? 

https://advocacy-mozilla-org-st-pr-504.herokuapp.com/en-US/privacynotincluded/

TheoChevalier commented on public/assets/buyers-guide/menu-burger.svg:
```diff
@@ -0,0 +1,21 @@
+                </g>
+            </g>
+            <text id="menu" font-family="OpenSans-Bold, Open Sans" font-size="10" font-weight="bold" letter-spacing="0.800000012">
+                <tspan x="0.46396482" y="35">menu</tspan>
```
Is there any way this string can be added into the template instead? Maybe displaying an element with some pseudo class?

gvn replied:
We could do that, but it won't render as consistently as SVG. Another option would be to remove the text. @beccaklam which would you prefer?

alanmoo replied:
Long shot, but using the l10n templating string doesn't work inside the SVG, does it?

TheoChevalier replied:
I would be very surprised if it does

beccaklam commented:
Yes! @gvn sorry I saw it this morning but got distracted. Will get on it right away.

beccaklam commented:
@gvn:
So some quick feedback:

- [ ] We should use the white mozilla logo for the menu:

![moz-logo-one-color-white-rgb](https://user-images.githubusercontent.com/20977376/35525902-df07c698-04f3-11e8-966b-7730402d7973.png)

- [ ] Can we center the menu content?  And add a margin-bottom of somewhere around 5-10px for each div.link-container?
<img width="1438" alt="screen shot 2018-01-29 at 1 12 37 pm" src="https://user-images.githubusercontent.com/20977376/35526676-3ef1949c-04f6-11e8-8d26-8f28cb8ddfd2.png">



- [ ] This is sort of unrelated but related to the landing page -- We need more padding on the left and right sides of the page, can we change:

> .buyers-guide-header-container {
> padding: 20px 45px; to padding: 20px 60px 24px
> }

and (actually not sure if this is the right selector but see image below)

> .buyers-guide-home .categories-container .home-category-container .category-top {
> padding: 0 45px to padding: 0 60px
> }

<img width="1440" alt="screen shot 2018-01-29 at 1 04 09 pm" src="https://user-images.githubusercontent.com/20977376/35526395-3c9930b6-04f5-11e8-961e-00d1408c4aa8.png">

gvn commented:
@beccaklam Ok, made some of those tweaks. I'm not sure what you mean by centering the menu content exactly.

Do you mean setting a max-width on the container or vertically centering it in the viewport? The comps look left aligned...

beccaklam commented:
@gvn Oh sorry I just meant centering the content. Setting the div that holds menu content to 

> margin: auto 0;

 seems to do it.
<img width="1437" alt="screen shot 2018-01-29 at 3 12 53 pm" src="https://user-images.githubusercontent.com/20977376/35532140-2e360f1e-0507-11e8-8957-fbdd3af3417e.png">

beccaklam commented:
^ also I am still seeing a black logo @gvn am I using a wrong staging link?

gvn commented:
You may have an older version cached. Try holding shift and clicking refresh.

So this is really what you want? Looks odd to me:

<img width="959" alt="screen shot 2018-01-29 at 1 05 33 pm" src="https://user-images.githubusercontent.com/206379/35534381-365f99ec-04f5-11e8-82e2-a9be4c28a50b.png">

beccaklam commented:
@gvn haha it's switched around:

> margin: auto 0;

instead of

> margin: 0 auto;

beccaklam commented:
@gvn also shift+refresh did not work. it's also still black in your last screen cap? i'm talking about the Mozilla logo in the full screen menu

gvn commented:
Ok, the filter wasn't working properly x-browser, but should be fixed now. I also tweaked the margin.

https://advocacy-mozilla-org-st-pr-504.herokuapp.com/en-US/privacynotincluded/

beccaklam commented:
@gvn Mobile view is looking good but needs some tightening up:
Current state:
![screen shot 2018-01-29 at 4 44 39 pm](https://user-images.githubusercontent.com/20977376/35536185-2086264e-0514-11e8-8549-95d7f811107b.png)

- [ ] The favicon should be reversed -- white text on black square (see comps)

- [ ] privacy not included should be left aligned with the asterisk sitting outside like so (not sure how you would do this in code but if you need an svg or png I can export one for you):
![category page](https://user-images.githubusercontent.com/20977376/35536360-a1d5e5b8-0514-11e8-8cc7-e23e5057c795.jpg)

gvn commented:
Made those tweaks...

beccaklam commented:
@gvn One more menu design QA:
Can we shift the X over to where I've drawn the red X in the following image:
<img width="376" alt="screen shot 2018-01-30 at 12 04 19 pm" src="https://user-images.githubusercontent.com/20977376/35579980-089d6c66-05b6-11e8-84f8-a5cdb779bf61.png">

gvn commented:
Ok, tweaks made. @beccaklam can you take another look at staging?

beccaklam commented:
@gvn Just looked. Soooo much better. Sorry I made a mistake:

- [ ] For the class "bg-subhead", font-size should be 12px not 23px for the mobile media query

gvn commented:
K...yeah, thought that looked weird.

beccaklam commented:
@gvn and 

- [ ] *privacy not included can be one line on desktop screens like before (just want mobile stacked):
<img width="513" alt="screen shot 2018-01-30 at 1 44 29 pm" src="https://user-images.githubusercontent.com/20977376/35584683-bf357826-05c3-11e8-975f-b4c06cc45b8d.png">

- For mobile screens can we 

- [ ] change the font size of *privacy not included to 16px and the line-height to 16.75px;

After that we should be good to ship! Thank you so much 💯

gvn commented:
Ok, I removed the forced break between "privacy" and "not included".

I added an additional breakpoint below 400px to reduce to 12px because it was getting really crowded at 16px.

beccaklam commented:
@gvn Design QA:

# **P1**

**Universal**
- [ ] We want to get rid of the Playfair typeface entirely. Can you replace all Playfair instances with Zilla Slab? 

- Menu on Product Pages (this relates to the above)

- [ ] 	change Playfair font to Open Sans

# **P2**

**Universal**
- [ ] 	change “ *privacy not included” font-weight from 900 to 400; 

**Mobile**
- All Pages
- [ ] “ *privacy not included ” should be stacked (with “not included” under “privacy”) BUT ONLY on mobile 

**Tablet/Desktop**
- [ ] Twitter icon font-size at tablet and desktop views should be at 24px to keep it consistent with the other social media icons (for some reason it’s just smaller than the other icons and that makes it look off) - P2

# **P3**

**Mobile**
- Product Page
- [ ] Can we change the related products divs to percentages so that they’re responsive? It looks strange that they’re stacked off to the left side. So somthing like this, you can play around with the numbers to see what works best:
<img width="374" alt="screen shot 2018-01-30 at 2 43 08 pm" src="https://user-images.githubusercontent.com/20977376/35589033-7896121a-05d0-11e8-9a92-e7187abd313c.png">

gvn commented:
Ok, made those changes minus the stacking of the logo text because I think that should adapt to the width of its parent container size instead of a breakpoint if we leave it as live responsive typography.

Alternatively, we can rasterize it as SVG in two different layouts, but that's going to be a pain for localization as new assets will need to be generated for every language.

beccaklam commented:
@gvn thanks changes look so good! I'm fine with leaving *privacy not included the way it is for now. Just some QA:

- [ ] The menu on product pages is still in Playfair. Again, probably some old janky styling where it's getting over ridden somewhere.
![screen shot 2018-01-31 at 10 30 16 am](https://user-images.githubusercontent.com/20977376/35632843-ccd6856c-0675-11e8-9703-9989cf252107.png)

- [ ] Found a bug on mobile where when the pop up comes because of the sticky header the user can't close the pop up and essentially is stuck on the page. This should probably get fixed:
<img width="375" alt="screen shot 2018-01-31 at 11 01 13 am" src="https://user-images.githubusercontent.com/20977376/35632950-1667fa26-0676-11e8-9512-d2d089e3d13a.png">

- [ ] I checked mobile for the related products and they're still sitting oddly to the left. Not a blocker to ship but would be nice to fix eventually.
![screen shot 2018-01-31 at 10 57 31 am](https://user-images.githubusercontent.com/20977376/35632877-e15caa98-0675-11e8-8ad0-70f3800b0dce.png)

Otherwise we're good to go 👍

alanmoo commented:
Merging, Rebecca will create a new ticket with remaining QA items.
